climbing_gym/demo/sale_order_line.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="sale_order_line_8" model="sale.order.line">
+        <field name="order_id" ref="sale_order_7"/>
+        <field name="product_id" ref="product_product_16"/>
+        <field name="name"><![CDATA[1 Entrance Happy Hour]]></field>
+        <field name="product_uom_id" ref="uom.product_uom_unit"/>
+        <field name="price_unit">12.0</field>
+        <field name="technical_price_unit">12.0</field>
```

Comment:
You can clean the code a bit here, since all these fields are retrieved from the product
```suggestion
```